app/config/config.yml.dist

```diff
@@ -380,6 +380,14 @@ cookies_lifetime: 1209600
 # to '.example.org'.
 cookies_domain:
 
+# By default Bolt stores the sessions according to what is configured in your
+# PHP settings. If this doesn't work correctly, you can override the settings
+# below. For a list of the available settings, see here:
+# https://github.com/bolt/bolt/blob/release/3.3/src/Provider/SessionServiceProvider.php#L152-L178
```

Comment:
That link also isn't static, as it points to `release/3.3` HEAD, and it is not unreasonable to expect it to change during the life time of `release/3.3`. It is already heavily rewritten in  both `release/3.4` & `master`. So someone installing a site on 3.3 now, and then a year from now working on 3.22.3, or 4.0.1 is going to be pointed there.

Another reason that this better belongs in the docs, in a more complete, and explained fashion. Not to mention, further demonstrates that there is too much already jammed in here to navigate … for all 3 groups.